src/components/Platform.jsx

```diff
@@ -13,24 +12,22 @@ function Points(props) {
   );
 }
 
-function Picture(props) {
+function Picture(name, image) {
```

Comment:
shouldn't do this, react components always take exactly 1 props arg

otherwise they won't work properly as JSX